index.html

```diff
@@ -1,20 +1,320 @@
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+            <div class="container-fluid">
+              <a class="navbar-brand" href="#">
+                <img
+                  src="./img/logo.png"
+                  alt="companyLogo"
+                  width="30"
+                  height="30"
+                  class="d-inline-block align-text-top"
+                />
+                Hevons Farms
+              </a>
+            </div>
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Products</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Services</a>
+              </li>
+            </ul>
+            <button
+              class="btn btn-outline-warning brand-btn ms-lg-5"
+              type="submit"
+            >
+              Partner
+            </button>
+          </div>
+        </div>
+      </nav>
+      <nav
+        class="bg-light container-fluid d-flex pt-2 px-5 d-none d-md-flex justify-content-end"
+        aria-label="breadcrumb"
+      >
+        <ol class="breadcrumb breadcrumb-space">
+          <li class="breadcrumb-item">
+            <a href="#"><span class="material-icons"> home_filled </span></a>
+          </li>
+          <li class="breadcrumb-item active text-primary" aria-current="page">
+            Library
+          </li>
+        </ol>
+      </nav>
+    </header>
+    <main class="bg-light">
+      <!-- Banner section -->
+      <section class="px-5 banner">
+        <h1 class="display-1 fw-bolder text-capitalize intro-txt">
+          Sustaining the future with food security today
+        </h1>
+        <div class="shadow-lg py-1 my-4 col-sm-12 col-md-12 col-lg-8">
+          <p class="fs-6 fw-light desc-txt my-1 mx-1 p-2">
+            Many countries are facing the double burden of hunger and
+            undernutrition alongside overweight and obesity, with one in three
+            people across the globe currently suffering from some form of
+            malnutrition. Indeed it is not unusual to find people with different
+            forms of malnutrition living side-by-side in one country, in one
+            community, or even in the same household.
+          </p>
+        </div>
+        <a
+          href="#"
+          class="display-3 text-capitalize text-decoration-none fs-1 fw-bold"
+          >join us today!</a
+        >
+        <p class="display-5 fs-5">
+          Together, let's secure a future for our generations yet to come
+        </p>
+      </section>
```

Comment:
- [ ] It would be great if you could add some more padding or margin to this section so as to match the original design.
- Your design:
![Capture1](https://user-images.githubusercontent.com/85113567/181268746-f720d862-547e-4f81-92d3-aeae20c5ab52.PNG)
- Required design:
![Capture2](https://user-images.githubusercontent.com/85113567/181268791-0f406b61-0116-48a2-8e74-2d4a182db139.PNG)

Reply:
resolved